apps/base/utils.py

```diff
@@ -85,7 +85,8 @@ def decode_data(data):
     """
     decoded = []
     for i in data:
-        decoded.append(base64.decodestring(i + "=="))
+        s = i + "=="
```

Comment:
Can you please remove this and add this directly in `decodebytes`
`decoded.append(base64.decodebytes((i + "==").encode()))`